sockeye/model.py

```diff
@@ -96,10 +96,11 @@ class SockeyeModel(mx.gluon.Block):
     time.
 
     :param config: Model configuration.
+    :param inference_only: Use the model only for inference, enabling optimizations.
     :param prefix: Name prefix for all parameters of this model.
     """
 
-    def __init__(self, config: ModelConfig, prefix: str = '', **kwargs) -> None:
+    def __init__(self, config: ModelConfig, inference_only: bool = False, prefix: str = '', **kwargs) -> None:
```

Comment:
thats a good addition. Can we move other optimization into here and bind them to the setting of that flag? For example `config.disable_dropout()`?

Reply:
yeah, makes sense to me.